### ISSUE TITLE
rat

### ISSUE THREAD

**Issue by clipflip-exploits** [werihes/kikima#1]
rat

**Comment by errorcraft-mc** [comment]
yep this is a rat made by solara. everything is a RAT. solara installs this to ur pc i will put solara source code on my github

**Comment by clipflip-exploits** [comment]
You stupid? this is a fake solara that installs this

**Comment by imjustfoxyy** [comment]
yes but, the real solara is no longer known. nobody knows what the real solara is now

**Comment by imjustfoxyy** [comment]
the solara rat from discord.gg/cscripts installs this
